### PR TITLE
!leave <role> command

### PR DESCRIPTION
**Pull request by adamschachne**

### CODE REVIEW

**Comment by adamschachne** [comment]
closes #23

**Comment by adamschachne** [comment]
Discord_2022-10-07_00-45-03

File: inhouse_bot/game_queue/queue_handler.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def add_player(
         session.merge(queue_player)
 
 
-def remove_player(player_id: int, channel_id: int = None):
+def remove_player(player_id: int, channel_id: int = None, role: str = None):
     """
     Removes the player from the queue in all roles in the channel
```
Comment (classBabacar) [inline inhouse_bot/game_queue/queue_handler.py]: We can change this comment since this functions sole purpose isnt this anymore.

**Review by classBabacar — CHANGES_REQUESTED** [review]

File: inhouse_bot/game_queue/queue_handler.py (hunk 1 above)
Comment (adamschachne) [reply inhouse_bot/game_queue/queue_handler.py]: good catch

**Review by classBabacar — APPROVED** [review]

**Pull request merged**